test/topology_custom/test_topology_smp.py

```diff
@@ -37,12 +38,27 @@ async def test_nodes_with_different_smp(request: FixtureRequest, manager: Manage
     # on both nodes, then it's guaranteed that the message will be
     # processed on the same shard as the calling code.
     # In the general case, we cannot assume that this same shard guarantee holds.
+
+    # The test is flaky on CI in debug builds on aarch64 (#14752),
+    # here we sprinkle more logs for debug/aarch64
+    # hoping it'll help to debug it.
+    # The hack with xmlpath: it's set by test.py to some file with mode in the path,
+    # couldn't think of a better way to determine the current mode.
+    # This should produce ~16M of logs per Scylla node.
+    log_args = []
+    if ('/debug/' in request.config.getoption('xmlpath', '')) and ('aarch64' in platform.processor()):
+        log_args = [
+            '--default-log-level', 'debug',
+            '--logger-log-level', 'raft_group0=trace:group0_client=trace:storage_service=trace'
+                                  ':raft=trace:raft_group_registry=trace'
+        ]
```

Comment:
This piece of code might be useful in other tests as well.
Perhaps extracting it to a fixture would make sense.
We can do this later, not in this PR.